libs/stylelint-config/package.json

```diff
@@ -15,18 +15,17 @@
   "dependencies": {
     "postcss-scss": "^4.0.9",
     "stylelint-config-property-sort-order-smacss": "^10.0.0",
-    "@stylistic/stylelint-plugin": "^2.1.2",
+    "@stylistic/stylelint-plugin": "^3.0.1",
     "stylelint-use-nesting": "^5.1.1",
     "stylelint-declaration-block-no-ignored-properties": "^2.8.0",
     "stylelint-file-max-lines": "^1.0.0",
-    "stylelint-scss": "^6.1.0",
+    "stylelint-scss": "^6.5.1",
     "stylelint-group-selectors": "^1.0.10"
```

Comment:
I tried to submit a pull request with the updates, but since I'm not a contributor to the project, I don't have the required permissions to commit to the repository. Fortunately, one of the existing contributors has created the PR, but it's currently pending approval from the original library author.